translator/tocwconfig/sampleConfig/complete_linux_config.yaml

```diff
@@ -145,6 +145,13 @@ processors:
               from_attribute: ""
               from_context: ""
               pattern: "telemetry.sdk.*"
+            - action: delete
```

Comment:
This should be here. I guess the other pr never added it. Now the unit tests pass.